Discovery/Web-Content/api/salesforce-aura-objects.txt

```diff
@@ -0,0 +1,1075 @@
+CollaborationGroup
+CollaborationGroupMember
+CollaborationGroupMemberRequest
+CollaborationGroupRecord
+CollaborationInvitation
+CollabDocumentMetric
+CollabDocumentMetricRecord
+CollabTemplateMetric
+CollabTemplateMetricRecord
+Collab
+Collab
+ColorDefinition
+CombinedAttachment
+CommerceEntitlementBuyerGroup
+CommerceEntitlementPolicy
+CommerceEntitlementPolicyShare
+CommerceEntitlementProduct
+CommissionSchedule
+CommissionScheduleAssignment
+CommSubscription
+CommSubscriptionChannelType
+CommSubscriptionConsent
+CommSubscriptionTiming
+Community
+ConnectedApplication
+ConsumptionRate
+ConsumptionSchedule
+Contact
+ContactCleanInfo
+ContactDailyMetric
+ContactMonthlyMetric
+ContactPointAddress
+ContactPointConsent
+ContactPointEmail
+ContactPointPhone
+ContactPointTypeConsent
+ContactOwnerSharingRule
+ContactRequest
+ContactRequestShare
+ContactShare
+ContactSuggestionInsight
+ContactTag
+ContentAsset
+ContentBody
+ContentDistribution
+ContentDistributionView
+ContentDocument
+ContentDocumen
+ContentDocumentLink
+ContentDocumentListViewMapping
+ContentDocumentSubscription
+ContentFolder
+ContentFolderItem
+ContentFolderLink
+ContentFolderMember
+ContentHubItem
+ContentHubRepository
+ContentNote
+ContentNotification
+ContentTagSubscription
+Content
+ContentVersion
+ContentVersionComment
+ContentVersionHistory
+ContentVersionRating
+ContentWorkspace
+ContentWorkspaceDoc
+ContentWorkspaceMember
+ContentWorkspacePermission
+ContentWorkspaceSubscription
+Contract
+ContractContactRole
+ContractLineItem
+ContractStatus
+ContractTag
+Conversation
+ConversationContextEntry
+ConversationEntry
+ConversationParticipant
+CorsWhitelistEntry
+Coupon
+CreditMemo
+CreditMemoLine
+Crisis
+CronJobDetail
+CronTrigger
+CspTrustedSite
+CurrencyType
+CustomBrand
+CustomBrandAsset
+CustomHelpMenuItem
+CustomHelpMen
+CustomHttpHeader
+CustomNotificationType
+CustomPermission
+CustomPermissionDependency
+Customer
+DandBCompany
+Dashboard
+DashboardComponent
+DashboardTag
+DataAssessmentFieldMetric
+DataAssessmentMetric
+DataAssessmentValueMetric
+DatacloudCompany
+DatacloudContact
+DatacloudDandBCompany
+DatacloudOwnedEntity
+DatacloudPurchaseUsage
+DataIntegrationRecordPurchasePermission
+DatasetExport
+DatasetExportPart
+Data
+Data
+DatedConversionRate
+DeclinedEventRelation
+DelegatedAccount
+DeleteEvent
+DigitalSignature
+DigitalWallet
+DirectMessage
+Division
+DivisionLocalization
+Document
+DocumentAttachmentMap
+DocumentTag
+Domain
+DomainSite
+DsarPolicy
+DsarPolicyLog
+DuplicateJob
+DuplicateJobDefinition
+DuplicateJobMatchingRule
+DuplicateJobMatchingRuleDefinition
+DuplicateRecordItem
+DuplicateRecordSet
+DuplicateRule
+ElectronicMediaGroup
+ElectronicMediaUse
+EmailContent
+EmailDomainFilter
+EmailDomainKey
+EmailMessage
+EmailMessageRelation
+EmailRelay
+EmailServicesAddress
+EmailServicesFunction
+EmailStatus
+EmailTemplate
+EmailTemplateMonthlyMetric
+EmbeddedServiceDetail
+EmbeddedServiceLabel
+Employee
+EmployeeCrisisAssessment
+Emp
+Emp
```

Comment:
Make sure to run `uniq` on this file to remove duplicates :p

Reply:
Removed duplicate values